corehq/apps/app_manager/static/app_manager/js/modules/module_view.js

```diff
@@ -153,9 +153,12 @@ hqDefine("app_manager/js/modules/module_view", function () {
         }
 
         // UCR last updated tile
-        $('#report-context-tile').koApplyBindings({
-            report_context_tile: ko.observable(moduleBrief.report_context_tile),
-        });
+        var $reportContextTile = $('#report-context-tile');
+        if ($reportContextTile.length) {
+            $reportContextTile.koApplyBindings({
+                report_context_tile: ko.observable(moduleBrief.report_context_tile),
+            });
+        }
```

Comment:
So the error in the ticket was just that after hitting this code right here, it raised an error which prevented other stuff from getting hooked up, and so the changes users made to the UI elements didn't affect the underlying js values?

Reply:
Yes, anything further down in this file didn't run and likely didn't save (I say "likely" because we do occasionally have area that use js but also degrade gracefully).